client/src/dataset/components/DatasetParams.tsx

```diff
@@ -16,12 +16,15 @@ const renderParamValue = (value: any) => {
 
 const renderParams = (params: any) => {
     return Object.keys(params).map((key: string, idx: number) => {
-        return (
-            <Table.Row key={idx}>
-                <Table.Cell>{key}</Table.Cell>
-                <Table.Cell>{renderParamValue(params[key])}</Table.Cell>
-            </Table.Row>
-        );
+        if(key !== "dataset_paths") {
```

Comment:
I've removed `dataset_paths` from the info as:
1. The `ds_path`s are shown as [[object Object], [object Object], ...]
2. All the `ds_path`s are already specified below (in the info box) individually

Reply:
Fixed!